### ISSUE TITLE
Не работает перезагрузка

### ISSUE THREAD

**Issue by PGSStas** [PGSStas/techvscats#118]

**Comment by anevero** [comment]
Конечно, надо проверить, но мы вообще и на систему полагаемся. Проблемы на ее стороне тоже могут возникнуть.
Но проверить надо.

**Comment by anevero** [comment]
Проблема на стороне системы. Переименуйте папку так, чтобы в ее названии не было пробелов. То ли QProcess, то ли Windows не могут корректно обработать путь к исполняемому файлу, содержащий пробелы.

**Comment by anevero** [comment]
На будущее - в тот `if` можно добавить выкидывание простенького QMessageBox с подписью "Возникла ошибка при запуске нового процесса", можно туда еще выводить путь, по которому попытались запустить. А то мало ли еще где-нибудь такая фигня всплывет.
